libs/MobileSync/MobileSync.xcodeproj/project.pbxproj

```diff
@@ -1391,6 +1391,7 @@
 				INFOPLIST_FILE = MobileSync/Info.plist;
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
+				"IPHONEOS_DEPLOYMENT_TARGET[sdk=macosx*]" = 14.5;
```

Comment:
Shouldn't this be `MACOS_DEPLOYMENT_TARGET`? It sounds like `IPHONEOS_DEPLOYMENT_TARGET` should be `13.0`, i.e. the line above.

Reply:
I was surprised that's what the diff was too, but here's what it looks like in the project settings. The change was setting macOS to 11.3
<img width="155" alt="xcode" src="https://user-images.githubusercontent.com/840431/117338260-33d44500-ae53-11eb-887c-e8af750218b5.png">